Assets/Scripts/Device/DiscordControls.cs

```diff
@@ -42,28 +42,28 @@ public class DiscordControls {
         if (details != "") {
             rpDetails = details;
         }
-		
-		if (time != 0) {
-			rpTime = ((int)(System.DateTime.UtcNow - epochStart).TotalSeconds) + time;
-		} else {
-			rpTime = 0;
-		}
-		
-		
-		var activityManager = discord.GetActivityManager();
+        
+        if (time != 0) {
+            rpTime = ((int)(System.DateTime.UtcNow - epochStart).TotalSeconds) + time;
+        } else {
+            rpTime = 0;
+        }
+        
+        
+        var activityManager = discord.GetActivityManager();
 
         var activity = new Discord.Activity
         {
             State = rpDetails,
             Details = rpName,
             Timestamps = {
                 Start = (remaining ? 0 : rpTime),
-				End = (remaining ? rpTime : 0)
+                End = (remaining ? rpTime : 0)
             },
-			Assets = {
-				LargeImage = "cyf_logo",
-				LargeText = "Create Your Frisk"
-			}
+            Assets = {
+                LargeImage = "cyf_logo",
+                LargeText = "Create Your Frisk"
+            }
         };
 
         activityManager.UpdateActivity(activity, (res) => {
```

Comment:
Try to add more info in these Debug messages: we can't know where they come from if all we see is "Failed" or "Everything is fine!".
Plus, if these Debug messages are displayed a lot, please remove them entirely.
We don't want the log file to be cluttered.